WordPress/Classes/Services/BlogService.m

```diff
@@ -185,9 +185,11 @@ - (void)syncOptionsForBlog:(Blog *)blog
                    failure:(void (^)(NSError *error))failure
 {
     id<BlogServiceRemote> remote = [self remoteForBlog:blog];
-    [remote syncOptionsWithSuccess:[self optionsHandlerWithBlogObjectID:blog.objectID
-                                                  completionHandler:success]
-                           failure:failure];
+    if ([remote respondsToSelector:@selector(syncOptionsWithSuccess:failure:)]) {
+        [remote syncOptionsWithSuccess:[self optionsHandlerWithBlogObjectID:blog.objectID
+                                                          completionHandler:success]
+                               failure:failure];
+    }
```

Comment:
If something other than `syncBlogs` calls this, it should also work for WordPress.com blogs. I think in that case this should sync site details.